lib/scrabble.rb

```diff
@@ -1,6 +1,8 @@
 class Scrabble
-  def point_values
-    {
+  attr_reader  :point_values, :caps
+
+  def initialize
+    @point_values = {
```

Comment:
Minor thing, but when you have a moment think through how you can use the hash inside of a method.